### PR TITLE
Clean imports

### PR DESCRIPTION
**Pull request by MartinHjelmare**
- Remove not used import.
- Add needed libraries. Not sure if we should add it to common requirements, or limit these somehow.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
+pillow
 pyramid
-pyramid_jinja2
\ No newline at end of file
+pyramid_jinja2
+sqlalchemy
```
Comment (oeway) [inline requirements.txt]: It seems we can remove this and also the file called `models.py` -- it's not used anywhere.

File: requirements.txt [hunk 2]
```diff
@@ -1,2 +1,4 @@
+pillow
```
Comment (oeway) [inline requirements.txt]: Where did we used pillow?

Comment (MartinHjelmare) [reply requirements.txt]: Here:
https://github.com/imjoy-team/jupyter-elfinder/blob/f0a4ad0e56b3c053b4bf80fbd1a16ccbc9b47229/jupyter_elfinder/elfinder.py#L1421

File: requirements.txt (hunk 1 above)
Comment (MartinHjelmare) [reply requirements.txt]: Sounds good!

Comment (MartinHjelmare) [reply requirements.txt]: Done.

**Comment by oeway** [comment]
Is this one ready?

**Comment by MartinHjelmare** [comment]
Yes, thanks!

**Pull request merged**